docs/getting_started.rst

```diff
@@ -22,6 +22,7 @@ Once Great Expectations is installed, follow this tutorial for a quick start.
 
    /getting_started/cli_init
    /getting_started/typical_workflow
+   /getting_started/command_line
```

Comment:
(Documenting our call): move this into its own top level menu TOC item under the expectations glossary